src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (num) {
+      return this.result + num;
+    },
+    subtract: function (num) {
+      return this.result - num;
```

Comment:
For the `subtract` method, use the `-=` operator to subtract from the existing value.

BAD EXAMPLE:
 return this.result - num;

GOOD EXAMPLE:
 this.result -= num;

Please update the `subtract` method accordingly.